server/status/recorder.go

```diff
@@ -55,10 +59,32 @@ func NewNodeStatusRecorder(monitor *NodeStatusMonitor, clock *hlc.Clock) *NodeSt
 	}
 }
 
+// recordInt records a single int64 value from the NodeStatusMonitor as a
+// proto.TimeSeriesData object.
+func (nsr *NodeStatusRecorder) recordInt(timestampNanos int64, name string,
+	data int64) proto.TimeSeriesData {
+	return proto.TimeSeriesData{
+		Name: fmt.Sprintf(nodeTimeSeriesNameFmt, name, nsr.nodeID),
+		Datapoints: []*proto.TimeSeriesDatapoint{
+			{
+				TimestampNanos: timestampNanos,
+				Value:          float64(data),
+			},
+		},
+	}
+}
+
 // GetTimeSeriesData returns a slice of interesting TimeSeriesData from the
 // encapsulated NodeStatusMonitor.
 func (nsr *NodeStatusRecorder) GetTimeSeriesData() []proto.TimeSeriesData {
 	data := make([]proto.TimeSeriesData, 0, nsr.lastDataCount)
+	// Record node stats.
+	if nsr.nodeID > 0 {
+		now := nsr.clock.PhysicalNow()
+		data = append(data, nsr.recordInt(now, "calls.successcount", atomic.LoadInt64(&nsr.callCount)))
```

Comment:
It wasn't a matter of future heirarchy, but rather clarity in the metric name - i've truncated them to `calls.success` and `calls.error`.